testutils/anvil.go

```diff
@@ -131,12 +131,7 @@ func GetContractAddressesFromContractRegistry(ethHttpUrl string) (mockAvsContrac
 }
 
 func AdvanceChainByNBlocks(n int, anvilEndpoint string) {
-	cmd := exec.Command("bash", "-c",
-		fmt.Sprintf(
-			// see https://book.getfoundry.sh/reference/anvil/#custom-methods
```

Comment:
I think the link to the anvil custom methods can be useful for the potential readers of this code

Reply:
Thanks, comment restored!